global/js/main.js

```diff
@@ -715,16 +715,44 @@ function getAlerts() {
 
 }
 
+function insertParam(key, value) {
+    if (!window.history.pushState) {
+        return;
+    }
+
+    if (!key) {
+        return;
+    }
+
+    var url = new URL(window.location.href);
+    var params = new window.URLSearchParams(window.location.search);
+    if (value === undefined || value === null) {

```

Comment:
better to check undefined with typeof
```suggestion
    if (typeof value == 'undefined' || value === null) {

```